src/service/viewport/viewport-impl.js

```diff
@@ -203,6 +203,15 @@ export class Viewport {
     }
     if (viewer.getParam('webview') === '1') {
       this.globalDoc_.documentElement.classList.add('i-amphtml-webview');
+      // Pages that have a <body> height of 0 (e.g. if content is in a
+      // position:absolute wrapper) will be hidden due to `overflow-x:hidden` in
+      // `i-amphtml-webview` CSS. Detect and fix by setting height to 100vh.
+      // See b/74541306.
+      this.ampdoc.whenBodyAvailable().then(body => {
+        if (body./*OK*/getBoundingClientRect().height == 0) {
+          setImportantStyles(body, {'height': '100vh'});
```

Comment:
Maybe `min-height: 100vh`? Also wondering why not just add this to `.i-amphtml-webview {...}` CSS in amp.css?

One note: I don't believe this will be fixed with v3 SD wrapper - wrapper is only used for iframes.

Reply:
Wanted fix to be as minimal as possible but that works too. Done.